### PR TITLE
fix(midways): FRMW-638-eliminate midway test warnings.

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-638

- [x] Dev LGTM
- [x] Dev LGTM
- [x] QE LGTM

---
**Tech Debt:** Eliminate warnings in midways tests.

### CODE REVIEW

File: src/components/rxButton/docs/rxButton.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <!-- Sample HTML goes here as a live example of how to the component can be used -->
 <h3>Typical Usage</h3>
 <div ng-controller="rxButtonCtrl">
-    <rx-button toggle-msg="Authenticating" default-msg="Login"
+    <rx-button id="demo-ui-rx-button" toggle-msg="Authenticating" default-msg="Login"
```
Comment (CITguy) [inline src/components/rxButton/docs/rxButton.html]: For readability and to adhere to HTML coding standards, please place attributes on separate, indented lines.

```html
<rx-button
  id="demo-ui-rx-button"
  default-msg="Login"
  toggle-msg="Authenticating"
  toggle="status.loading"
  ng-click="login()">
</rx-button>
```

File: src/components/rxButton/docs/rxButton.midway.js [hunk 2]
```diff
@@ -5,7 +5,7 @@ describe('rxButton', function () {
 
     before(function () {
         demoPage.go('#/components/rxButton');
-        rxButton = rxButtonPage.initialize($('rx-button'));
+        rxButton = rxButtonPage.initialize($('#demo-ui-rx-button'));
```
Comment (CITguy) [inline src/components/rxButton/docs/rxButton.midway.js]: * remove `rxButtonPage` variable from top of file
* replace `rxButtonPage` with `encore.rxButton`

File: src/components/rxPaginate/docs/rxPaginate.midway.js [hunk 3]
```diff
@@ -79,7 +79,7 @@ describe('rxPaginate', function () {
     describe('UI pagination exercises', encore.exercise.rxPaginate({
         pageSizes: [3, 50, 200, 350, 500],
         defaultPageSize: 3,
-        cssSelector: '.demo-ui-pagination .rx-paginate'
+        instance: encore.rxPaginate.initialize($('.demo-ui-pagination .rx-paginate'))
```
Comment (CITguy) [inline src/components/rxPaginate/docs/rxPaginate.midway.js]: :+1:

File: src/components/rxPaginate/docs/rxPaginate.html [hunk 4]
```diff
@@ -1,105 +1,115 @@
...
-                        sort-column="sort.predicate"
-                        sort-direction="sort.reverse">
-                    </rx-paginate>
-                </td>
-            </tr>
-        </tfoot>
-    </table>
+  <button ng-click="pagedServers.refresh()" class="button sm">Refresh</button>
+  <table rx-floating-header rx-loading-overlay class="table-striped demo-api-pagination">
+      <thead>
```
Comment (CITguy) [inline src/components/rxPaginate/docs/rxPaginate.html]: `thead`, `tbody`, and `tfoot` are indented too far. please correct.

File: src/components/rxSearchBox/scripts/rxSearchBox.page.js [hunk 5]
```diff
@@ -112,7 +112,7 @@ exports.rxSearchBox = {
     */
     main: (function () {
         rxSearchBox.rootElement = {
-            get: function () { return $('rx-search-box'); }
+            get: function () { return $$('rx-search-box').first(); }
```
Comment (100stacks) [inline src/components/rxSearchBox/scripts/rxSearchBox.page.js]: @Droogans, updated to resolve these warnings.....
screen shot 2016-02-03 at 11 09 30 am

File: src/components/rxPaginate/docs/rxPaginate.midway.js [hunk 6]
```diff
@@ -94,7 +94,7 @@ describe('rxPaginate', function () {
         var osColumn = table.column('OS');
 
         beforeEach(function () {
-            encore.rxFloatingHeader.scrollToElement(table.tblResults);
+            encore.rxMisc.scrollToElement(table.tblResults);
```
Comment (100stacks) [inline src/components/rxPaginate/docs/rxPaginate.midway.js]: Updated based on this deprecation warning: https://github.com/rackerlabs/encore-ui/blob/master/src/components/rxFloatingHeader/scripts/rxFloatingHeader.page.js#L10

File: src/components/rxPaginate/docs/rxPaginate.html (hunk 4 above)
Comment (100stacks) [inline src/components/rxPaginate/docs/rxPaginate.html]: @CITguy, updated indentation. :+1:

File: src/components/rxSearchBox/scripts/rxSearchBox.page.js (hunk 5 above)
Comment (Droogans) [inline src/components/rxSearchBox/scripts/rxSearchBox.page.js]: I'm torn. Part of me wants those warnings to appear...if you're throwing default selectors at a page with multiple instances of the same component, you're doing something wrong. You should see a warning.

However, the warning itself is bit harsh for something so innocuous (if you know what you're doing, that is).

File: src/components/rxPaginate/docs/rxPaginate.midway.js (hunk 6 above)
Comment (Droogans) [inline src/components/rxPaginate/docs/rxPaginate.midway.js]: Thanks!

File: src/components/rxButton/docs/rxButton.html [hunk 7]
```diff
@@ -1,17 +1,26 @@
-<!-- Sample HTML goes here as a live example of how to the component can be used -->
 <h3>Typical Usage</h3>
 <div ng-controller="rxButtonCtrl">
-    <rx-button toggle-msg="Authenticating" default-msg="Login"
-        toggle="status.loading" ng-click="login()">
-    </rx-button>
+  <rx-button
+    id="demo-ui-rx-button"
```
Comment (CITguy) [inline src/components/rxButton/docs/rxButton.html]: `id` attribute added in addition to indenting existing attributes

**Comment by CITguy** [comment]
LGTM

File: src/components/rxSearchBox/scripts/rxSearchBox.page.js (hunk 5 above)
Comment (100stacks) [inline src/components/rxSearchBox/scripts/rxSearchBox.page.js]: @Droogans, no problem! As I understand the path forward, you and @CITguy are working on refactoring `EncoreUI` page objects anyways.

**Comment by parlarjb** [comment]
LGTM

**Pull request merged**